arcgis-ios-sdk-samples/Layers/Display KML/DisplayKMLViewController.swift

```diff
@@ -0,0 +1,169 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+
+class DisplayKMLViewController: UIViewController {
+    
+    @IBOutlet weak var mapView: AGSMapView!
+    
+    /// A model for our three data sources.
+    enum KMLDataSource: String, CaseIterable {
```

Comment:
If you change the raw type from `String` to `Int` then you have access to an index for the current `displaySource`. This would mean that you wouldn't have to iterate through each cell to find the one with a `reuseIdentifier` that matches `displayedSource`'s raw value.

This change would simplify the code's readability and remove the need for iterating through all cells.